web/server/vue-cli/src/assets/userguide/userguide.md

```diff
@@ -295,9 +295,14 @@ run history events.
 - filters will be saved in the URL query parameter list.
 
 ### Compare runs
+
 Calculates the difference between two analyses of the project, showing which
 reports have been fixed and which are newly introduced.
 
+:warning: Note: Here, you can read about _how_ to query the differences in
+between two results set. As to how this comparison works, refer to the docs
+on [`CodeChecker cmd diff`](../../../../../../docs/web/user_guide.md#show-differences-between-two-runs-diff).
```

Comment:
Maybe we should use absolute paths in documentations. That would be more resilient for potential moves of the current doc.

Reply:
Do you know how to do that though? :)

Reply:
```
[`CodeChecker cmd diff`](/docs/web/user_guide.md#show-differences-between-two-runs-diff)
```

Reply:
Now I realized, that this is a documentation which is displayed in the GUI. These paths may not be valid there anyway, so feel free to skip this comment of mine. I'm not even sure, that this "../../../" version works, because `user_guide.md` is not the part of the GUI. Maybe we could provide a link directly to this doc on GitHub.